src/test/java/DukeTest.java

```diff
@@ -0,0 +1,45 @@
+import org.junit.jupiter.api.Test;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+/**
+ * DukeTest class is a test class for Duke class.
+ */
+public class DukeTest {
+
+    /**
+     * Tests the deadline's toString method for the valid format.
+     */
+    @Test
+    public void testDeadline() {
+        LocalDate date = LocalDate.parse("2020-02-02", DateTimeFormatter.ofPattern("yyyy-MM-dd"));
```

Comment:
in larger tests, you may want to move these setup steps to separate methods to avoid code duplication. check what does @Before and @After do in JUnit.